Source/JavaScriptCore/disassembler/zydis/Zydis/ZycoreFormat.c

```diff
@@ -34,7 +34,7 @@
 #if defined(ZYAN_CLANG)
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wcast-qual"
-#pragma clang diagnostic ignored "-Wformat-nonliteral"
+ALLOW_NONLITERAL_FORMAT_BEGIN
```

Comment:
If this comes from Zycore-C should we be changing it?

Reply:
Also, it becomes inconsistent with the others.lines above.

Reply:
I'll revert this file.